src/ui/server/helpers/mappings/map-sic-codes.ts

```diff
@@ -1,12 +1,13 @@
 import { SicCode } from '../../../types';
 
 /**
- * maps through database response for sicCode and returns array of strings
+ * maps through database response for sicCode andindustrySectorName and returns array of strings
  * @param {Array<SicCode>} codes
+ *  @param {String} type what to return from sicCode array
```

Comment:
It's not clear to me what `type` is? Can we better describe it? Also indentation